.lintstagedrc.json

```diff
@@ -11,6 +11,6 @@
     "git add"
   ],
   "*.md": [
-    "remark -qf ."
+    "remark -q ."
```

Comment:
Could you tell us why the `--frail` option has been removed?

Reply:
with --frail, the commit was failing because of warnings

Reply:
i added it back, now its working, not sure what changed